fix: description and hint of Learn Accessibility by Building a Quiz - Step 41

SwapnilThatte opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #49521

<!-- Feel free to add any additional description of changes below this line -->

SwapnilThatte committed:
Update in description 

Updated instruction in Learn Accessibility by Building a Quiz - Step 41, added new instruction " The submit should display the text "Send". "

SwapnilThatte committed:
Merge branch 'freeCodeCamp:main' into main

SwapnilThatte committed:
Update In description and hint

1) Changed description to " Do not forget to give your form a submit button with the text "Send". "
2) Added word "button" in the hint

SwapnilThatte committed:
Merge pull request #1 from SwapnilThatte/SwapnilThatte-patch-1

Update In description and hint

SwapnilThatte committed:
Merge branch 'freeCodeCamp:main' into main

jeremylt commented on curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145f685797bd30df9784e8c.md:
```diff
@@ -7,7 +7,7 @@ dashedName: step-41
 # --description--
 
-Do not forget to give your `form` a submit button.
+Do not forget to give your form a submit button with the text "Send".
```
`form` is a code element name. I don't think we should change this?

```suggestion
Do not forget to give your `form` a submit button with the text "Send".
```

Sboonny replied:
The word Send shouldn't be translatable as well

```suggestion
Do not forget to give your `form` a submit button with the text `Send`.
```

SwapnilThatte committed:
Merge branch 'freeCodeCamp:main' into main

SwapnilThatte committed:
Update in learn-accessibility-by-building-a-quiz

Updated  learn-accessibility-by-building-a-quiz Step 41, updated text "form" to " `form` "

SwapnilThatte committed:
Merge pull request #2 from SwapnilThatte/SwapnilThatte-patch-1

Update in learn-accessibility-by-building-a-quiz

SwapnilThatte committed:
Update in learn-accessibility-by-building-a-quiz

Replaced "submit" with "submit button".

SwapnilThatte commented:
I have made the changes in all the hints, please verify.

Sboonny commented on curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145f685797bd30df9784e8c.md:
```diff
@@ -7,7 +7,7 @@ dashedName: step-41
 # --description--
 
-Do not forget to give your `form` a submit button.
+Do not forget to give your `form` a submit button with the text "Send".
```
```suggestion
Do not forget to give your `form` a submit button with the text `Send`.
```

SwapnilThatte committed:
Changed "Send" to `Send`

Sboonny reviewed (approved):
LGTM 🎉, thank you for your contribution to the page! 👍.

jeremylt reviewed (approved):
LGTM
